rpm/0081-sailfishos-webrtc-Implement-video-capture-module.-JB.patch

```diff
@@ -1,21 +1,21 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Denis Grigorev <d.grigorev@omprussia.ru>
 Date: Thu, 4 Feb 2021 21:14:28 +0300
-Subject: [PATCH] [sailfishos][webrtc] Implement video capture module. JB#53982
+Subject: [PATCH] Implement video capture module. JB#53982
```

Comment:
```suggestion
Subject: [PATCH] [sailfishos][webrtc] Implement video capture module. JB#53982
```
Please keep changelog components around. It helps when checking git log with patches applied given that you applied patches with "git am --keep-non-patch"

Reply:
Thanks for the tip. Fixed.